Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -3703,6 +3703,7 @@ end
 				constructor =
 				{
 					{
+						Returns = { {Type = "cCompositeChat"} },
```

Comment:
Yes. But I don't want to force it in the simulator, because this way I can find the problematic exports